philippa-choong/week2/tue/index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+     <link rel="stylesheet" href="css/style.css"> 
+    <title>Tim Berners-Lee Home Page</title>
+</head>
+
+<body>
+    <div class="container"> 
+        <!-- <nav class = "navbar">
```

Comment:
You need to uncomment out line 13, just uncomment line 14, thats why your nav bars are different widths.

Reply:
you should only one nav bar. 
You should change this to `div class="navbar"` and then it will be:
`<div>`
       `       <nav>`
      `       </nav>`
`</div>`